SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc

```diff
@@ -275,6 +275,7 @@ HcalDigitizer::HcalDigitizer(const edm::ParameterSet& ps, edm::ConsumesCollector
 
   edm::ParameterSet ps0 = ps.getParameter<edm::ParameterSet>("HcalReLabel");
   relabel_ = ps0.getUntrackedParameter<bool>("RelabelHits");
+//  std::cout << "Flag to see if Hit Relabeller to be initiated " << relabel_ << std::endl;
```

Comment:
I just want to make sure that this PR is fully a new code, in a sense that there is no other PR introducing overlapping changes.
Please confirm.